pkg/products/observability/reconciler.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
 	prometheus "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	observability "github.com/redhat-developer/observability-operator/v3/api/v1"
+	//observability1 "github.com/valerymo/observability-operator/v3/api/v1"
```

Comment:
I think we can remove this import as it looks to be duplicated :thinking:

Reply:
Done. Thank you